Documentation/ceph-cluster-crd.md

```diff
@@ -815,12 +815,25 @@ So just taking the `storage` section this will give something like:
           volumeMode: Block
           accessModes:
             - ReadWriteOnce
+      - metadata:
```

Comment:
If we are going to add this example to the documentation, we should also describe when it should be used. To keep the current example of the metadata device simpler, we should likely create a new example instead of adding to this example.